apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -332,6 +332,8 @@ function initializeBlocklyWrapper(blocklyInstance) {
   };
   blocklyWrapper.JavaScript = javascriptGenerator;
   blocklyWrapper.navigationController = new NavigationController();
+  // Initialize plugin.
+  blocklyWrapper.navigationController.init();
```

Comment:
Maybe worth a comment that this means it's called once per page load, while `inject` can now potentially be called more than once?  

Though I'm not actually sure whether folks arriving at this code in the future will wonder...

Reply:
Hmm yeah I can update this comment